nl_NL/LC_MESSAGES/localeSelection.po

```diff
@@ -1,17 +1,17 @@
 msgid "CHANGE_LANGUAGE"
-msgstr "Change Language"
+msgstr "Verander Taal (Change Language)"
```

Comment:
Please omit the Change Language bit -- I don't think it will fit in the menu.
If someone mistakenly changes their language, I think they should be able to figure out where they went originally to change it -- also considering that it will display "Nederlands" in the confirmation dialog, and "Nederlands" in the box in the Creator's Room settings as well, which will be in the same place.

Reply:
Good point! I'll just change it real fast...